layouts/mapbuilder/component.jsx

```diff
@@ -0,0 +1,153 @@
+      bgImage={bgImage}
+    />
+    <div className="summary-section">
+      <div className="row">
+        {summary.map((item) => (
+          <div key={item.title} className="column small-12 medium-6 large-3 ">
+            <div className="summary-card">
+              <Icon icon={item.icon} className="summary-icon" />
+              <h5 className="summary-title">{item.title}</h5>
+              <p className="summary-description">{item.summary}</p>
+            </div>
+          </div>
+        ))}
+      </div>
+    </div>
+    <div className="guide-section">
+      <div className="row">
+        <div className="column small-12">
+          <h4 className="guide-title">Start building your GFW MapBuilder</h4>
+        </div>
+        <div className="column small-12">
+          <p className="guide-intro">
+            Here&apos;s your 1-2-3 guide on getting started.
+          </p>
+        </div>
+        {guide.map((item, index) => (
+          <div key={item.key} className="column small-12 medium-4 guide-column">
+            <div className="guide-card">
+              <span className="guide-number">{index + 1}</span>
+              <p className="guide-text">{item.text}</p>
+            </div>
+          </div>
+        ))}
+        <div className="column small-10 small-offset-1 medium-4 medium-offset-4 large-2 large-offset-5">
+          <Link href="https://www.globalforestwatch.org/help/mapbuilder/">
+            <a>
+              <Button className="column-btn">get started</Button>
+            </a>
+          </Link>
+        </div>
+      </div>
+    </div>
+    <div className="maps-section">
+      <div className="row">
+        <div className="column small-12">
+          <h4 className="maps-title">Featured maps</h4>
+        </div>
+        <div className="column small-12">
+          <p className="maps-intro">
+            A hand-picked selection of GFW maps and applications built by our
+            user community. Imagine your map or app here!
+          </p>
+        </div>
+        {maps.map((item) => (
+          <div key={item.title} className="column small-12 medium-4 maps-card">
+            <a href={item.extLink} target="_blank" rel="noopener noreferrer">
+              <Card theme="theme-card-small" data={item} />
+            </a>
+          </div>
+        ))}
+        <div className="column small-12">
+          <div className="row">
+            <div className="column small-10 small-offset-1 medium-3 medium-offset-3">
+              <a href="mailto:gfw@wri.org">
+                <Button className="column-btn" theme="theme-button-light">
+                  submit your map
+                </Button>
+              </a>
+            </div>
+            <div className="column small-10 small-offset-1 medium-3 medium-offset-0">
+              <Link href="https://www.globalforestwatch.org/help/mapbuilder/">
```

Comment:
View all redirects to help center, if this is the case I would recommend some other copy so we are not misleading, even just reusing the above sections "get started" is fine